apps/hash-api/src/graphql/resolvers/knowledge/entity/entity.ts

```diff
@@ -479,10 +482,19 @@ export const addEntityViewerResolver: ResolverFn<
 > = async (_, { entityId, viewer }, { dataSources, authentication }) => {
   const context = dataSourcesToImpureGraphContext(dataSources);
 
-  await addEntityViewer(context, authentication, {
-    entityId,
-    viewer: parseGqlAuthorizationViewerInput(viewer),
-  });
+  await modifyEntityAuthorizationRelationships(context, authentication, [
+    {
+      operation: "create",
```

Comment:
```suggestion
      operation: "touch",
```

Reply:
Thank you, applied in [`7e4aa7f` (#3423)](https://github.com/hashintel/hash/pull/3423/commits/7e4aa7f959fd2357bdfd9489cfa36ca5bae3a2db)